pkg/iac/scanners/dockerfile/scanner_test.go

```diff
@@ -553,27 +552,23 @@ res := true
 
 	for _, tc := range testCases {
 		t.Run(tc.name, func(t *testing.T) {
-			regoMap := make(map[string]string)
-			libs, err := rego.LoadEmbeddedLibraries()
-			require.NoError(t, err)
-			for name, library := range libs {
-				regoMap["/rules/"+name] = library.String()
-			}
```

Comment:
OPA has a bug with converting a module to a string and back. https://github.com/open-policy-agent/opa

Especially there is no point in loading the libraries yourself, as Rego scanner can do it.